zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -17,7 +17,7 @@
 
 angular.module('zeppelinWebApp')
   .controller('ParagraphCtrl', function($scope,$rootScope, $route, $window, $element, $routeParams, $location,
-                                         $timeout, $compile, websocketMsgSrv) {
+                                         $timeout, $compile, websocketMsgSrv, Notification) {
```

Comment:
@djoelz : Run the build then it will find it.
OR run `bower install` in zeppelin-web